homeassistant/components/lg_thinq/strings.json

```diff
@@ -43,6 +43,256 @@
       "sabbath_mode": {
         "name": "Sabbath"
       }
+    },
+    "select": {
+      "wind_strength": {
+        "name": "Speed",
+        "state": {
+          "slow": "Slow",
+          "low": "Low",
+          "mid": "Medium",
+          "high": "High",
+          "power": "Turbo",
```

Comment:
Who do we have 2 Turbo?

Reply:
Because we have device variations, some devices use 'power' for 'Turbo' and other devices use 'turbo' for 'Turbo'.